composer.json

```diff
@@ -28,8 +28,8 @@
     "require": {
         "php": "^7.2",
         "dealerdirect/phpcodesniffer-composer-installer": "^0.5.0",
-        "slevomat/coding-standard": "dev-master#469368cc6d8fe83aceba259ef65f1c0a2f63055b",
-        "squizlabs/php_codesniffer": "^3.5.0"
+        "slevomat/coding-standard": "^6.0",
+        "squizlabs/php_codesniffer": "^3.5"
```

Comment:
Please lock at 
```suggestion
        "squizlabs/php_codesniffer": "^3.5.3"
```

Reply:
Ok. I though it's sufficient that 3.5.3 is resolved through slevomat/cs and for us 3.5 will generally do.